.gitignore

```diff
@@ -1,3 +1,4 @@
 _site/
 .sass-cache/
 .jekyll-metadata
+*~
```

Comment:
I didn't realize that everything got pushed at once, and the pull request was just all the files.